src/main/java/org/apache/maven/plugins/war/util/PathSet.java

```diff
@@ -46,21 +47,16 @@
     /**
      * Set of normalized paths
      */
-    private Set<String> pathsSet = new LinkedHashSet<String>();
+    private Set<String> pathsSet = new LinkedHashSet<>();
 
-    /**
-     * The method normalizes the path.
-     * <ul>
-     * <li>changes directory separator to unix's separator(/)</li>
-     * <li>deletes all trailing slashes</li>
-     * </ul>
-     *
-     * @param path to normalization
-     * @return normalized path
-     */
-    protected String normalizeFilePath( String path )
+    static String normalizeSubPath( String path )
     {
-        return normalizeFilePathStatic( path );
+        if ( path.isEmpty() )
+        {
+            return path;
+        }
+        final String cleanPath = Paths.get( StringUtils.replace( path, '\\', '/' ) ).toString();
```

Comment:
Here we are assuming that 'path' is an absolute path or it is relative to the current directory of the JVM, is this true?
Won't we have problems with mounts/payh traversals...?
It seems to be that previous code was only operating on strings, not on filesystem.
Do I miss something?